2020-03-04-devpeople-meetup.yml

```
@@ -0,0 +1,5 @@
+name: 'Meetup #1 | Пользовательские интерфейсы | DevPeople'
```

Предлагаю упростить название до такого
```suggestion
name: 'DevPeople meetup #1'
```